omf/compat/omf_v1.py

```diff
@@ -9,9 +9,23 @@
 import numpy as np
 import properties
 
+from ..attribute import (
+    CategoryAttribute,
+    CategoryColormap,
+    ContinuousColormap,
+    NumericAttribute,
+    StringAttribute,
+    VectorAttribute,
```

Comment:
Direct imports mean you get errors earlier when changing things, helping to avoid bugs in production.